release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1181,6 +1185,7 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Bare Metal Hardware Provisioning*
 
+* Previously, the machine instance `state` annotation was not set. Consequently, the `STATE` column was empty. With this update, the machine instance `state` annotation is now set, and information in the `STATE` column automatically populates. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1857008[*BZ#1857008*])
```

Comment:
It looks like you had this one in the Bug Fix Summary spreadsheet, but crossed-out the Final Bug Description. Any reason why the matter is addressed here and not in the bug fixes?